modules/plottinggl/include/modules/plottinggl/processors/parallelcoordinates/parallelcoordinates.h

```diff
@@ -194,8 +195,9 @@ class IVW_MODULE_PLOTTINGGL_API ParallelCoordinates : public Processor {
 
         // startFilter, startRegular, startSelected, end
         std::array<size_t, 4> offsets;
-
+        
         std::vector<float> axisPositions;
+        std::vector<float> axisFlipped;
```

Comment:
float vector? Should be bool? Otherwise comment why

Reply:
Changed to `std::vector<int>` instead. Using int here for performance reasons since bool is not supported as GLSL uniform. A bool vector would internally be converted to an int array prior setting the uniform.

see UniformSetter<std::array<bool, N>>